Css/WebScrap.Css/Contracts/ICssComparer.cs

```diff
@@ -0,0 +1,15 @@
+using WebScrap.Css.Data;
+using WebScrap.Core.Tags;
+
+namespace WebScrap.Css.Contracts;
+
+public interface ICssComparer
```

Comment:
please write summaries on all public stuff in this project.
It's the root for understanding.
Just a brief explanation, a few words.